ppa/archive/templates/archive/snippets/search_result.html

```diff
@@ -83,9 +84,15 @@
                         {% include 'snippets/chevron.svg' %}
                     </div>
                 </div>
+
             </div>
         </div>
     </a>
+    {# if the cluster id is set and differs from item id, work is part of a cluster #}
+    {# only display when not searching within a cluster  #}
+    {% if not search_form.cluster.value and item.cluster_id and item.cluster_id != item.id %}
+        {% include "archive/snippets/cluster_work.html" with cluster_id=item.cluster_id %}
+    {% endif %}
```

Comment:
Just flagging that these line relates to other issue with cluster id being set in any case, even when there is no cluster.